### DISCUSSION TITLE
Bug Report: German Translation "Marken" -> "Mark"

### DISCUSSION THREAD

**Discussion by Michael-Zinn** [FakeFishGames/Barotrauma#14499] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The German translation uses "Marken" when talking about money. It should always be "Mark", as in "10 Mark", not "10 Marken".

"Mark" is the currency, it doesn't get a plural, similar to "10 Dollar" or "10 Euro".

"Marken" is used when talking about tokens or food stamps ("Essensmarken"). "10 Marken" reads like "10 food stamp tokens" in German.

I think doing a global replace of "Marken" to "Mark" in the translation files should be safe, unless the game also mentions other tokens/stamps anywhere.

### Reproduction steps

1. Set language to German
2. Look at money amounts

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

MacOS

**Comment by Regalis11** [comment]
Thank you for the report! I've reported this issue to the localization team, who will (most likely) fix it for the next update.
